CMakeLists.txt

```diff
@@ -73,6 +73,16 @@ if (CMAKE_BUILD_TYPE STREQUAL "")
     set( CMAKE_BUILD_TYPE "RelWithDebInfo" )
 endif()
 
+option(ENABLE_WSTRING "Enable std::wstring support" ON)
```

Comment:
Change to `POCO_ENABLE_WSTRING`, I will merge next time the pull request #1437 so POCO properties are in one group in cmake....